pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -92,6 +92,15 @@ type DesiredStateOfWorld interface {
 	// attached to this node and the pods they should be mounted to based on the
 	// current desired state of the world.
 	GetVolumesToMount() []VolumeToMount
+
+	// GetDesiredPodVolumeSpecNames generates and returns a map of pods in which map
+	// is indexed with pod uid and value is a map of the pod's mounted volume spec
+	// names. This map can be used to determine which pod and volume is currently
+	// in desired state of world based on pod uid and volume spec name.
+	GetDesiredPodVolumeSpecNames() map[string]map[string]bool
+
+	// GetVolumePluginMgr returns the volume plugin manager
+	GetVolumePluginMgr() *volume.VolumePluginMgr
```

Comment:
Let's get rid of this. `reconciler.go` should not access `volumePluginMgr` through the `desired_state_of_world`. If it needs the  Mgr it should be explicitly passed in the reconciler constructor.